commands/errors/handling.go

```diff
@@ -0,0 +1,89 @@
+package errors
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/gsctl/client/clienterror"
+	"github.com/giantswarm/gsctl/oidc"
+)
+
+// HandleCommonErrors is a common function to handle certain errors happening in
+// more than one command. If the error given is handled by the function, it
+// prints according text for the end user and exits the process.
+// If the error is not recognized, we simply return.
+//
+func HandleCommonErrors(err error) {
+
+	var headline = ""
+	var subtext = ""
+
+	// V2 client error handling
+	if convertedErr, ok := microerror.Cause(err).(*clienterror.APIError); ok {
+		headline = convertedErr.ErrorMessage
+		subtext = convertedErr.ErrorDetails
+	} else if convertedErr, ok := err.(*clienterror.APIError); ok {
+		headline = convertedErr.ErrorMessage
+		subtext = convertedErr.ErrorDetails
+	} else {
+		// legacy client error handling
+		switch {
+		case oidc.IsAuthorizationError(err):
+			headline = "Unauthorized"
+			subtext = "Something went wrong during a OIDC operation: " + err.Error() + "\n"
+			subtext += "Please try logging in again."
```

Comment:
I'd like to see use of `fmt.Sprintf` here to keep this tidy in one line. Optional change as it's just cleaning up legacy bits

Reply:
Done